components/grid/templates/column-chooser.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Column Chooser
+page_title: Grid - Column Chooser Template
+description: Customize the content of the Column Chooser.
+slug: grid-templates-column-chooser
+tags: telerik,blazor,grid,templates,column,menu
+published: True
+position: 40
+---
+
+# Column Chooser Template
+
+The `Column Chooser Template` lets you control the rendering of the [ColumnChooser]({%slug grid-column-menu%}#column-chooser). It exposes a `context` object that contains a List of all columns in the Grid. 
+
+>note When the Template is setup the list of columns, rendered by default, would not be present.
```

Comment:
Does not have to be a note

Reply:
I agree, I was wondering which was the better option.